src/aspire/image/image.py

```diff
@@ -341,7 +341,8 @@ def _im_translate(self, shifts):
 
         # Note original stack shape and flatten stack
         stack_shape = self.stack_shape
-        im = self.stack_reshape(-1)._data
+        # upcast to np.float64 to preserve numerical stability in FFT ops
+        im = self.stack_reshape(-1)._data.astype(np.float64)
```

Comment:
see my other remarks. This file can be reverted to `origin/develop` if you take that other patch.